src/Tizen.NUI.Components/Style/PaginationStyle.cs

```diff
@@ -106,7 +106,7 @@ public Selector<string> IndicatorImageUrl
         /// <since_tizen> 8 </since_tizen>
         public int IndicatorSpacing
         {
-            get => ((int?)GetValue(IndicatorSpacingProperty)) ?? 0;
+            get => (int)GetValue(IndicatorSpacingProperty);
```

Comment:
If the `IndicatorSpacingProperty` returns null, this code would cause crash.
I think the API should have been `int?` type but it is `int` type and ACRed :(
There is no choice but to use `?? 0`.

Reply:
Ah, i agree with that. i will keep `?? 0`.